Android.mk

```diff
@@ -98,7 +98,27 @@ ifeq ($(strip $(BOARD_DISABLE_NATIVE_COLOR_MODES)),true)
 LOCAL_CPPFLAGS += -DDISABLE_NATIVE_COLOR_MODES
 endif
 
-ifeq ($(strip $(BOARD_USES_VULKAN)),)
+ifeq ($(strip $(BOARD_USES_VULKAN)),true)
+LOCAL_SHARED_LIBRARIES += \
+        libvulkan
```

Comment:
I think this file has been using tabs instead of spaces, might as well be consistent...